pkgs/by-name/ll/lldap-cli/package.nix

```
@@ -0,0 +1,56 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, bash
+, coreutils
+, gnugrep
+, gnused
+, jq
+, curl
+, makeWrapper
+}:
+stdenv.mkDerivation {
+  pname = "lldap-cli";
+  version = "nightly-2024-02-24";
+
+  src = fetchFromGitHub {
+    owner = "Zepmann";
+    repo = "lldap-cli";
+    rev = "d1fe50006c4a3a1796d4fb2d73d8c8dcfc875fd5";
+    hash = "sha256-ZKRTYdgtOfV7TgpaVKLhYrCttYvB/bUexMshmmF8NyY=";
+  };
+
+  buildInputs = [ bash coreutils gnugrep gnused jq curl ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  patchPhase = ''
+    # fix .lldap-cli-wrapped showing up in usage
+    substituteInPlace lldap-cli \
+      --replace-fail '$(basename $0)' lldap-cli
+  '';
```

please keep pre/post-phase hooks. same idea with `installPhase`.
```suggestion
  patchPhase = ''
    runHook prePatch

    # fix .lldap-cli-wrapped showing up in usage
    substituteInPlace lldap-cli \
      --replace-fail '$(basename $0)' lldap-cli

    runHook postPatch
  '';
```